decidim-admin/spec/commands/decidim/admin/hide_resource_spec.rb

```diff
@@ -52,10 +52,19 @@ module Decidim::Admin
 
     context "when the resource is already hidden" do
       let(:moderation) { create(:moderation, reportable: reportable, report_count: 1, hidden_at: Time.current) }
+      let(:authors) { reportable.try(:authors) || [reportable.try(:author)] }
+      let(:reasons) { reportable.moderation.reports.pluck(:reason).uniq }
 
       it "broadcasts invalid" do
         expect { command.call }.to broadcast(:invalid)
       end
+
+      it "sends email to author" do
+        Decidim::Admin::HiddenResourceMailer.notify_mail(reportable, authors, reasons).deliver_now
+
+        expect(last_email.subject).to eq(I18n.t("decidim.admin.hidden_resource_mailer.notify_mail.subject"))
+        expect(last_email.to).to eq(authors.pluck(:email).uniq)
+      end
```

Comment:
```suggestion

      it "sends email to author" do
        clear_enqueued_jobs
        command.call

        expect(ActionMailer::MailDeliveryJob).to have_been_enqueued.on_queue("mailers")
        queued_user, _, queued_options = ActiveJob::Arguments.deserialize(ActiveJob::Base.queue_adapter.enqueued_jobs.first[:args]).last[:args]

        expect(queued_user).to eq(current_user)
      end
```
In here we want to test if a mail has been sent by the command, we cannot send the mail manually in the test as this wouldn't check what we want.
Instead, we should tell the test to run the pending jobs (created by the command), this is done with the helper  `clear_enqueued_jobs`. The rest is a trick to check if it worked